samples/subsys/usb/shell/sample.yaml

```diff
@@ -18,5 +18,5 @@ tests:
     extra_args: CONF_FILE="device_and_host_prj.conf"
                 OVERLAY_CONFIG="virtual.conf"
                 DTC_OVERLAY_FILE="virtual.overlay"
-    platform_allow: qemu_cortex_m3
```

Comment:
OK, I will fix this way, for now I will remove commits in question and make separate PR. There is benefit for adding more qemu type, at least for qemu_x86_64, as it found some print format issues.

Reply:
@jfischer-no I have removed this commit for now from PR

Reply:
> This actually not right (my fault), there are board specific overlays, correct fix would be to remove OVERLAY_CONFIG and DTC_OVERLAY_FILE args first, then add board specific overlays for qemu_x86 qemu_x86_64 (btw, I do not see any benefit to add additional qemu type).

Removed as you have recommended since the test was failing otherwise.